ipmof/energymap.py

```diff
@@ -110,6 +122,47 @@ def energy_map_atom_index(atom_name, emap_atom_list):
     return int(emap_atom_index + 3)
 
 
+def export_energy_map(emap, atom_list, sim_par, sim_dir, mof_name):
```

Comment:
You usually want to have your save file routine just accept what you're saving and the file name, and handle any specific file naming code a level up. So here you would just pass a filename, and you would put together the filename where this method is being called. That gives you the option in the future of saving files to different places.